### DISCUSSION TITLE
Install

### DISCUSSION THREAD

**Discussion by Ghostsniper89** [gicamm/homeassistant-comelit#20] [General]
Hello
I saw your wonderful plugin and I was wondering: can you tell me how to install it in the home assistant that I didn't understand?  What works with your plugin? 

Thanks a lot and nice job

**Comment by Ghostsniper89** [comment]
Hi

**Comment by gicamm** [comment]
I'm sorry for the considerable delay.
Something wrong happened with notifications.

Basically, you have to put the "comelit" folder under the _custom_components_ folder.
Then you have to enable the configuration and restart home assistant.

Did you look at the [wiki](https://github.com/gicamm/homeassistant-comelit/wiki)
?
Please let me know if I can help you.
